docs/oscar_references.bib

```diff
@@ -1746,6 +1679,14 @@ @Article{Sta79
   year          = {1979}
 }
 
+@Misc{Stacks,
+  bibkey        = {Stacks},
+  author        = {{A}uthors, {T}he {S}tacks {P}roject},
```

Comment:
The braces lead to all kinds of confusion, and they are not necessary either (unlike BibTeX, there is no automatic change in capitalization being performed here).

Reply:
The thesis by Johannes Schmitt, and the bib item right after it, also use extra braces in the title, which are rendered badly in this PR, so they should go for both

Reply:
Stacks and OEIS are still on my todo-list.
For the braces, I certainly missed some, thanks for pointing out